### ISSUE TITLE
date modelization

### ISSUE THREAD

**Issue by eroux** [buda-base/owl-schema#147]
I'm fine with the new bedate, csdate, etc. so it's just a proposal:

from my point of view, we could discriminate between two types of dates:
- modern dates (things that we can encode as an `xsd:date`
- the rest, which constitutes most of the date indications in the artefactual sources (Buddhist Era, Tibetan dates, the myriads of Chinese eras, etc.)

so as a rule of thumb, for a date indication, we could encode both: 
- the modern date would be used for queries, and would be always present when it's reasonably knowable
- the artefactual date would be indicated just like in the text

Now, it seems to me that in order to record a date indication we could start with just two fields:
- `:era` having a range of an era object
- `:yearInEra` which indicates the year

This would work for Chinese dates (if we encode all the various eras), and the SEA dates (the Tibetan dates would be a little different but they're a bit easier).

So in the end my proposal is that in the case where the artefact indicates `123 CS` and we know that it corresponds to 345CE, I would encode it in the following manner:
```
:onDate 345 ;
:dateIndication [
   :era bdr:EraCS ;
   :yearInEra 123
] .
```

what do you think?

**Comment by eroux** [comment]
Here's an attempt that also encodes Chinese eras:

https://docs.google.com/document/d/1MhEXs8hxBjYp6qXrmNIKGFyhMbKa53YOjhDJCeUdxTI/edit#

and a proposal to encode the full date indications (not just the era / year), including the Chinese ones:

https://docs.google.com/document/d/1dUO5Yb3Fs9yp_6Y1HGJ-Vw7nFghFuRwhYlowSln6QJo/edit#

what do you think?

**Comment by eroux** [comment]
the current option is to have:

```
“123”^^bdo:cdDate
```

**Comment by xristy** [comment]
yes that's the original desired intent and it turns out to require a bit of implementation which has been added to `io.bdrc.libraries.BdrcDateType` and integrated into `xmltoldmigration` and the `fuseki` startup
